test/release/examples/primers/LAPACKlib.chpl

```diff
@@ -1,46 +1,57 @@
 /*
-  Example usage of the LAPACK module in Chapel. This particular file
-  demonstrates how to use the gesv procedure with Chapel's arrays. The gesv
-  procedure solves A*X = B for X given both A and B, where A is a square matrix.
+  Example usage of the LAPACK module in Chapel.
 
-  Here, we create A and X, then compute B with a matrix multiplication, and
+  This particular file demonstrates how to use the gesv procedure with Chapel's
+  arrays. The gesv procedure solves ``A*X = B`` for ``X`` given both ``A`` and ``B``, where
+  ``A`` is a square matrix.
+
+  Here, we create ``A`` and ``X``, then compute ``B`` with a matrix multiplication, and
   show that the result of the gesv procedure is the same as our synthesized
-  X array (within a small margin of error for floating point).
+  ``X`` array (within a small margin of error for floating point).
 
   To compile a program with LAPACK, you may need to add some additional flags
   depending on how LAPACK was installed on your system:
 
-  chpl -I$PATH_TO_LAPACK_INCLUDE_DIR \
-       -L$PATH_TO_LAPACK_BINARIES \
-       -lgfortran
+  .. code-block:: sh
+
+      chpl -I$PATH_TO_LAPACK_INCLUDE_DIR \
+           -L$PATH_TO_LAPACK_BINARIES \
+           -lgfortran
 
   See the LAPACK module documentation for more information on compiling.
```

Comment:
It'd be nice if we could :ref: to [Compiling LAPACK](http://chapel.cray.com/docs/master/modules/packages/LAPACK.html#compiling-with-lapack)